antarest/core/utils/utils.py

```diff
@@ -61,6 +61,24 @@ def sanitize_uuid(uuid: str) -> str:
     return uuid
 
 
+def validate_study_name(name: str) -> str:
+    if any(char in name for char in ["=", "/"]):
+        raise HTTPException(
+            status_code=http.HTTPStatus.BAD_REQUEST, detail=f"study name {name} contains illegal characters (=, /)"
+        )
+    return name.strip()
+
+
+def validate_folder_name(name: str) -> str:
```

Comment:
Actually here we don't validate the name but the full path, we should probably rename it to make it clear.